testing/php7-pdlib/APKBUILD

```diff
@@ -0,0 +1,35 @@
+# Contributor: goodspb <goodspb.luo@gmail.com>
+# Maintainer: Adriaan Groenenboom <agboom@pm.me>
+pkgname=php7-pdlib
+_pkgreal=pdlib
+pkgver=1.0
+pkgrel=0
+pkgdesc="PHP extension for Dlib"
+url="https://github.com/goodspb/pdlib"
+arch="all"
+license="MIT"
+depends="php7-common dlib"
+makedepends="php7-dev pkgconf"
+source="$pkgname-$pkgver.tar.gz::https://github.com/goodspb/$_pkgreal/archive/v$pkgver.tar.gz"
+builddir="$srcdir/$_pkgreal-$pkgver"
+
+build() {
+	cd "$builddir"
```

Comment:
this lines no longer needed in every function

Reply:
Thanks for your feedback @andypost. The linter warned me about this too, but when I remove them, the build doesn't seem to succeed anymore. I'll try it again.

Reply:
It seems that it works without these lines, so I'll remove them in the next commit.